tensorflow/lite/micro/tools/make/targets/cortex_m_qemu_makefile.inc

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+export PATH := $(DOWNLOADS_DIR)/gcc_embedded/bin/:$(PATH)
+TARGET_TOOLCHAIN_PREFIX := arm-none-eabi-
+
+DOWNLOAD_RESULT := $(shell $(MAKEFILE_DIR)/arm_gcc_download.sh ${DOWNLOADS_DIR} $(TENSORFLOW_ROOT))
+ifneq ($(DOWNLOAD_RESULT), SUCCESS)
+  $(error Something went wrong with the GCC download: $(DOWNLOAD_RESULT))
+endif
+
+DOWNLOAD_RESULT := $(shell $(MAKEFILE_DIR)/ext_libs/cmsis_download.sh  ${DOWNLOADS_DIR} $(TENSORFLOW_ROOT))
+ifneq ($(DOWNLOAD_RESULT), SUCCESS)
+  $(error Something went wrong with the CMSIS download: $(DOWNLOAD_RESULT))
+endif
+
+PLATFORM_FLAGS = \
+  -DTF_LITE_MCU_DEBUG_LOG \
+  -mcpu=$(TARGET_ARCH) \
+  -mthumb \
+  -mfloat-abi=soft \
+  -mfpu=auto \
+  -funsigned-char \
+  -mlittle-endian \
+  -fomit-frame-pointer \
+
+# Enable semihosting for QEMU to enable host system calls for debug_log and micro_time.
+PLATFORM_FLAGS += --specs=rdimon.specs
+# USE_SEMIHOSTING enables host system calls for debug_log and micro_time.
+# Please see micro/cortex_m_generic micro_time.cc and debug_log.cc for more.
+PLATFORM_FLAGS += -DUSE_SEMIHOSTING
```

Comment:
this is not needed.

When `TARGET=cortex_m_qemu`,  we will end up using: https://github.com/tensorflow/tflite-micro/blob/2e2dfae3f6ee355649feb6ecc9dd6b13fd60ab21/tensorflow/lite/micro/micro_time.cc#L37-L47

and https://github.com/tensorflow/tflite-micro/blob/2e2dfae3f6ee355649feb6ecc9dd6b13fd60ab21/tensorflow/lite/micro/debug_log.cc#L43-L50

since there is no `micro/cortex_m_qemu` directory. We can fine tune what we want micro_time to be doing for qemu later, and fprintf worked just fine with qemu + semihosting.

Here is the logic if you're interested: https://github.com/tensorflow/tflite-micro/blob/2e2dfae3f6ee355649feb6ecc9dd6b13fd60ab21/tensorflow/lite/micro/tools/make/Makefile#L584-L588